wetterdienst/dwd/radar/index.py

```diff
@@ -103,15 +104,28 @@ def _create_fileindex_radar(
         files_server, columns=[DWDMetaColumns.FILENAME.value], dtype="str"
     )
 
+    # Some directories have both "---bin" and "---bufr" files within the same directory,
```

Comment:
Are both needed or would bin for all be fine?

Reply:
I don't know if it's needed. I just found @meteoDaniel started the `RadarDataType` enumeration und after getting into the details of data myself, I found e.g. https://opendata.dwd.de/weather/radar/sites/px/boo/ has **both** data types within the same directory. So, I figured the respective routine for filtering the outcome based on the obtained `radar_data_type` argument would be appropriate here.

I believe the PR in general just cares about **properly accessing and indexing** the different locations where radar data is stored for now. When it comes to supporting the actual **download**, `collect_radar_data` only handles RADOLAN through `_collect_radolan_data`, so the acquisition of alternative radar data beyond RADOLAN has not made it into the highlevel API `DWDRadarRequest` yet. There's more to come here and I will try to continue working on that.